src/Executables/Examples/HelloWorld/SingletonHelloWorld.hpp

```diff
@@ -29,15 +29,11 @@ struct Name {
 /// [executable_example_action]
 namespace Actions {
 struct PrintMessage {
-  template <typename DbTags, typename... InboxTags, typename Metavariables,
-            typename ArrayIndex, typename ActionList,
-            typename ParallelComponent>
+  template <typename ParallelComponent, typename DbTags, typename Metavariables,
+            typename ArrayIndex>
   static void apply(db::DataBox<DbTags>& /*box*/,
-                    tuples::TaggedTuple<InboxTags...>& /*inboxes*/,
                     const Parallel::ConstGlobalCache<Metavariables>& cache,
-                    const ArrayIndex& /*array_index*/,
-                    const ActionList /*meta*/,
-                    const ParallelComponent* const /*meta*/) {
+                    const ArrayIndex& /*array_index*/) {
```

Comment:
Because I had to change simple actions anyway and this makes the argument list easier to read and deal with. Yes, it could be done with iterable actions but I don't have time right now.